deploy/migrations/clickhouse/034_libp2p_gossipsub_beacon_block.up.sql

```diff
@@ -0,0 +1,80 @@
+CREATE TABLE libp2p_gossipsub_beacon_block_local on cluster '{cluster}' (
+    event_date_time DateTime64(3) Codec(DoubleDelta, ZSTD(1)),
+    slot UInt32 Codec(DoubleDelta, ZSTD(1)),
+    slot_start_date_time DateTime Codec(DoubleDelta, ZSTD(1)),
+    epoch UInt32 Codec(DoubleDelta, ZSTD(1)),
+    epoch_start_date_time DateTime Codec(DoubleDelta, ZSTD(1)),
+    wallclock_slot UInt32 Codec(DoubleDelta, ZSTD(1)),
+    wallclock_slot_start_date_time DateTime Codec(DoubleDelta, ZSTD(1)),
+    wallclock_epoch UInt32 Codec(DoubleDelta, ZSTD(1)),
+    wallclock_epoch_start_date_time DateTime Codec(DoubleDelta, ZSTD(1)),
+    propagation_slot_start_diff UInt32 Codec(ZSTD(1)),
+    block FixedString(66) Codec(ZSTD(1)),
+    proposer_index UInt32 CODEC(ZSTD(1)),
+    peer_id_unique_key Int64,
+    message_id String CODEC(ZSTD(1)),
+    message_size UInt32 Codec(ZSTD(1)),
+    topic_layer LowCardinality(String),
+    topic_fork_digest_value LowCardinality(String),
+    topic_name LowCardinality(String),
+    topic_encoding LowCardinality(String),
+    meta_client_name LowCardinality(String),
+    meta_client_id String Codec(ZSTD(1)),
+    meta_client_version LowCardinality(String),
+    meta_client_implementation LowCardinality(String),
+    meta_client_os LowCardinality(String),
+    meta_client_ip Nullable(IPv6) Codec(ZSTD(1)),
+    meta_client_geo_city LowCardinality(String) Codec(ZSTD(1)),
+    meta_client_geo_country LowCardinality(String) Codec(ZSTD(1)),
+    meta_client_geo_country_code LowCardinality(String) Codec(ZSTD(1)),
+    meta_client_geo_continent_code LowCardinality(String) Codec(ZSTD(1)),
+    meta_client_geo_longitude Nullable(Float64) Codec(ZSTD(1)),
+    meta_client_geo_latitude Nullable(Float64) Codec(ZSTD(1)),
+    meta_client_geo_autonomous_system_number Nullable(UInt32) Codec(ZSTD(1)),
+    meta_client_geo_autonomous_system_organization Nullable(String) Codec(ZSTD(1)),
+    meta_network_id Int32 Codec(DoubleDelta, ZSTD(1)),
+    meta_network_name LowCardinality(String)
+) Engine = ReplicatedMergeTree('/clickhouse/{installation}/{cluster}/tables/{shard}/{database}/{table}', '{replica}')
+PARTITION BY toStartOfMonth(slot_start_date_time)
+ORDER BY (slot_start_date_time, meta_network_name, meta_client_name);
+
+CREATE TABLE libp2p_gossipsub_beacon_block on cluster '{cluster}' AS libp2p_gossipsub_beacon_block_local
+ENGINE = Distributed('{cluster}', default, libp2p_gossipsub_beacon_block_local, rand());
+
+ALTER TABLE libp2p_gossipsub_beacon_block_local ON CLUSTER '{cluster}'
```

Comment:
move this one above `libp2p_gossipsub_beacon_block` so the comments are copied when the distributed table gets created